feat: reset baseline powertool

nbey opened the pull request:
- powertool that allows admins to reset the baseline to the previous levels PL, on current student competencies.

nbey committed:
feat: reset baseline powertool

themightychris commented on site-root/cbl/powertools/reset-baselines.php:
```diff
@@ -0,0 +1,173 @@
+use Slate\People\Student;
+
+
+$GLOBALS['Session']->requireAccountLevel('Developer');
```
I usually open powertools to Admins+

```suggestion
$GLOBALS['Session']->requireAccountLevel('Administrator');
```

themightychris commented on site-root/cbl/powertools/reset-baselines.php:
```diff
@@ -0,0 +1,173 @@
+}
+
+$contentArea = null;
+if (isset($_REQUEST['content_area'])) {
```
does only using isset instead of !empty work when one of the filters is submitted with no value?

themightychris reviewed (changes requested):
awesome work so far

nbey committed:
feat: improve UI / calculation

- use higher of previous PL / baseline
- add select all checkbox
- show previous PL/baselines
- add update to: column
- wrap checkbox in labels

nbey committed:
feat: allow admins to use

nbey committed:
refactor: use `empty` checks instead of `isset`
